src/main/java/com/inf2b/projectcodegeneration/services/TransactionService.java

```diff
@@ -0,0 +1,29 @@
+package com.inf2b.projectcodegeneration.services;
+
+import com.inf2b.projectcodegeneration.jwt.JwtTokenProvider;
+import com.inf2b.projectcodegeneration.models.Transaction;
+import com.inf2b.projectcodegeneration.models.User;
+import com.inf2b.projectcodegeneration.repositories.TransactionRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TransactionService {
+
+    private final TransactionRepository transactionRepository;
+
+    public TransactionService(TransactionRepository transactionRepository){
+        this.transactionRepository = transactionRepository;
+    }
+
+    public Transaction getAll(){
```

Comment:
```suggestion
    public List<Transaction> getAll(){
```
't Is een get all dus je krijgt een list terug

Reply:
Hahahaha die had ik niet gezien